.github/workflows/rollback.yml

```diff
@@ -140,12 +156,12 @@ jobs:
           mkdir -p ${{ env.SSM_ARTIFACT_ROOT }}
           for region in ${{ matrix.regions }}; do
             document_versions=$(aws ssm list-document-versions --name ${SSM_RELEASE_PACKAGE_NAME} --region ${region})
```

Comment:
Might be good to check the binaries that are released in S3 associate with SSM before checking version SSM thought.

Reply:
Added it.